Assets/Samples/5 - Initial Prompt/InitialPromptDemo.cs

```diff
@@ -0,0 +1,147 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using UnityEngine;
+using UnityEngine.UI;
+// ReSharper disable ArrangeObjectCreationWhenTypeEvident - for Unity 2019/2020 support:
+
+namespace Whisper.Samples
+{
+    public class InitialPromptDemo : MonoBehaviour
+    {
+        [Serializable]
+        public class InitialPrompt
+        {
+            public string name;
+            public string prompt;
+        }
+        
+        public WhisperManager manager;
+        public AudioClip clip;
+        public bool echoSound = true;
+
+        public List<InitialPrompt> initialPrompts = new List<InitialPrompt>
+        {
+            new InitialPrompt
+            {
+                name = "lowercase",
+                prompt = "hello how is it going always use lowercase no punctuation goodbye one two three start stop i you me they",
+            },
+            new InitialPrompt
+            {
+                name = "Start of the clip",
+                prompt = "And so my fellow Americans, ask not what your country can do for you",
+            },
+            new InitialPrompt
+            {
+                name = "UPPERCASE",
+                prompt = "EVERY WORD IS WRITTEN IN CAPITAL LETTERS AS IF THE CAPS LOCK KEY WAS PRESSED",
```

Comment:
This prompt for some reason doesn't work for me, but this one does.
"HELLO HOW IS IT GOING ALWAYS USE UPPERCASE NO PUNCTUATION GOODBYE ONE TWO THREE START STOP I YOU ME THEY"